frontend/src/components/advanced-auto-filter/advanced-auto-filter.tsx

```diff
@@ -57,6 +58,7 @@ const AdvancedAutoFilter: FC<AdvancedAutoFilterProps> = (props) => {
 
   const doSearch = (): void => {
     search(queryParams);
+    scroll.scrollToTop({ duration: 700 });
```

Comment:
I don't think that we need a separate dependency for just the scrolling.
We can simply use the build-it [`window.scroll`](https://developer.mozilla.org/en-US/docs/Web/API/Window/scroll) method

Reply:
@oleksandr-danylchenko because "smooth" scrolling doesn't work in `window.scroll({
      top: 0,
      left: 0,
      behavior: "smooth"
    })`

Reply:
Whoa, no way :thinking:  
![image](https://user-images.githubusercontent.com/68850090/188091770-6d8607ad-9abc-46b7-beee-04722552ee5d.png)

Reply:
Could you try the `scrollTo` method?